tests/testthat/test-api.R

```diff
@@ -945,6 +945,83 @@ test_that("api endpoint_download_coarse_output_head returns headers only", {
   expect_equal(res$body, "")
 })
 
+test_that("endpoint_download_summary can be run", {
+  test_redis_available()
+  test_mock_model_available()
+
+  queue <- test_queue(workers = 1)
+  run_endpoint <- endpoint_model_submit(queue)
+  path <- setup_submit_payload()
+  run_response <- run_endpoint$run(readLines(path))
+  expect_equal(run_response$status_code, 200)
+  out <- queue$queue$task_wait(run_response$data$id)
+
+  endpoint <- endpoint_download_summary(queue)
+  response <- endpoint$run(run_response$data$id)
+
+  expect_equal(response$status_code, 200)
+  expect_match(
+    response$headers$`Content-Disposition`,
+    'attachment; filename="MWI_\\d+-\\d+_summary_report.zip"')
+  size <- length(response$data)
+  expect_equal(response$headers$`Content-Length`, size)
+  expect_equal(size, file.size(system_file("dummy_summary_report.html")))
+
+  ## HEAD endpoint
```

Comment:
This looks like it's testing the wrong thing? (coarse inputs vs summary)

Reply:
Thanks, updated that